### DISCUSSION TITLE
VS code Live Share擴充功能

### DISCUSSION THREAD

**Discussion by pierre0210** [1101-GS4719-C/Python-Community-GS4719-C#707] [General]
## 簡介

如果想要請其他人幫忙看code或協作，除了用版控和丟PR外也可以使用[Live Share](https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare)這個擴充套件，它可以讓VS code有像Google文件那樣的協作功能。

## 使用方法
1. 安裝[Live Share](https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare)
2. 點擊左側Live Share圖示
![image](https://user-images.githubusercontent.com/54491348/192084590-9c36a0ed-7573-46f5-8cc1-1c3914e23ea0.png)
3. 按Share等待產生邀請連結
4. 協作者也需要安裝此擴充 點擊Join貼上邀請連結即可

**Comment by Yunhans** [comment]
我還沒試過這個
但看起來比git pull request即時
缺點可能會是看不懂其他人寫的程式吧
不知道有沒有回朔的功能

**Comment by danwan1027** [comment]
原來還有這個功能，以後請教別人問題時又更方便了！

**Comment by williamyen0623** [comment]
太酷了吧！看來vscoe無所不能！
